Module imports are not working

When trying to import symbols from any other TS module within an embark project that based on this template, webpack isn't able to resolve the module paths.

Here's how this can be reproduced:

```
$ embark new test --template embark-framework/embark-typescript-template
```
Then, take the `Greeter` class from `app/js/index.ts` and move it into `app/js/greeter.ts`:

```
// greeter.ts
export class Greeter {
  ...
}
```
Next try to import `Greeter` from this new file inside `index.ts`:

```
import { Greeter } from './greeter'
```

This will result in:

```
./app/js/index.ts
Module not found: Error: Can't resolve './greeter' in '/Users/pascalprecht/projects/embark/playground/ts-test/app/js'
resolve './greeter' in '/Users/pascalprecht/projects/embark/playground/ts-test/app/js'
  using description file: /Users/pascalprecht/projects/embark/playground/ts-test/package.json (relative path: ./app/js)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /Users/pascalprecht/projects/embark/playground/ts-test/package.json (relative path: ./app/js/greeter)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter doesn't exist
      .wasm
        Field 'browser' doesn't contain a valid alias configuration
        /Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter.wasm doesn't exist
      .mjs
        Field 'browser' doesn't contain a valid alias configuration
        /Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter.mjs doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter.js doesn't exist
      .json
        Field 'browser' doesn't contain a valid alias configuration
        /Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter.json doesn't exist
      as directory
        /Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter doesn't exist
[/Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter]
[/Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter.wasm]
[/Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter.mjs]
[/Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter.js]
[/Users/pascalprecht/projects/embark/playground/ts-test/app/js/greeter.json]
 @ ./app/js/index.ts 5:0-36 13:18-25
 @ multi ./app/js/index.ts
```

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -147,6 +147,16 @@ const base = {
   }
 };
 
+// typescript mods
+// -----------------------------------------------------------------------------
+base.resolve.extensions = [
+  // webpack defaults
+  // see: https://webpack.js.org/configuration/resolve/#resolve-extensions
+  '.wasm', '.mjs', '.js', '.json',
+  // typescript extensions
+  '.ts', '.tsx'
+];
+
 // development config
 // -----------------------------------------------------------------------------
 

